addons/mail/static/src/core/common/composer.js

```diff
@@ -164,7 +164,7 @@ export class Composer extends Component {
                     extraClass: "o-mail-Composer-dropzone",
                     onDrop: this.onDropFile,
                 },
-                () => this.allowUpload
+                () => this.props.allowUpload
```

Comment:
What is the purpose of this change? I tried to revert it and all the tests in `comments_popover` are still green.

Isn't it going backwards compared to the rest of the diff?

It seems we want to use `this.allowUpload` to take overrides into account rather than using from `props`.

I'd revert this unless we have a good reason to do it.